application/app/controllers/upload_files_controller.rb

```diff
@@ -71,24 +71,24 @@ def destroy
 
   def cancel
     project_id = params[:project_id]
-    collection_id = params[:upload_collection_id]
+    upload_batch_id = params[:upload_batch_id]
     file_id = params[:id]
 
-    if project_id.blank? || collection_id.blank? || file_id.blank?
+    if project_id.blank? || upload_batch_id.blank? || file_id.blank?
       render json: t(".compulsory_fields_error"), status: :bad_request
       return
     end
 
-    file = UploadFile.find(project_id, collection_id, file_id)
+    file = UploadFile.find(project_id, upload_batch_id, file_id)
 
     if file.nil?
-      render json: t(".file_not_found", project_id: project_id, collection_id: collection_id, file_id: file_id), status: :not_found
+      render json: t(".file_not_found", project_id: project_id, collection_id: upload_batch_id, file_id: file_id), status: :not_found
       return
     end
 
     if file.status.uploading?
       command_client = Command::CommandClient.new(socket_path: ::Configuration.command_server_socket_file)
-      request = Command::Request.new(command: 'upload.cancel', body: {project_id: project_id, collection_id: collection_id, file_id: file_id})
+      request = Command::Request.new(command: 'upload.cancel', body: {project_id: project_id, collection_id: upload_batch_id, file_id: file_id})
```

Comment:
I guess we need to refactor the request parameter name as well from collection_id to upload_batch_id